cabal.project

```diff
@@ -187,11 +187,15 @@ source-repository-package
   location: https://github.com/input-output-hk/hedgehog-extras
   tag: 8bcd3c9dc22cc44f9fcfe161f4638a384fc7a187
 
+source-repository-package
```

Comment:
I think drop this from the benchmarks. We don't need it for `Intmap+Levels`, and it gives us another awkward dependency.